website/templates/profile/user_settings_default.mako

```diff
@@ -1,29 +1,32 @@
 <!-- Authorization -->
 <div class="addon-oauth"
      data-addon-short-name="${ addon_short_name }"
-     data-addon-name="${ addon_full_name }">
-    <a data-bind="click: connectAccount" class="pull-right text-primary">Connect an account</a>
-    <h4 class="addon-title">{{ properName }}</h4>
+     data-addon-name="${ addon_full_name }">  
+    <h4 class="addon-title">
+      <img class="addon-icon" src="${addon_icon_url}"></img>
+      <span data-bind="text:properName"></span>
+      <small>
+        <a data-bind="click: connectAccount" class="pull-right text-primary">Connect Account</a>
+      </small>
+    </h4>
     <!-- ko foreach: accounts -->
-
-
     <table class="table">
         <thead>
             <tr>
-                <th>Authorized as <a href="{{ profileUrl }}"><em>{{ name }}</em></a></th>
-                <th><a data-bind="click: $root.askDisconnect" class="text-danger">Delete Access Token</a></th>
+                <th>Authorized by <a data-bind="attr.href: profileUrl, text: name"></a></th>
```

Comment:
The name here used to be italicized, which I think is preferred. Can you add it back?